iso/build.sh

```diff
@@ -17,37 +17,46 @@
 set -e
 
 ISO=minishift-b2d.iso
+curdir=$(pwd)
 tmpdir=$(mktemp -d)
+[ "${REMOVE_CONTAINER_IMAGES+1}" ] && DOCKER_RUN_OPTIONS="--rm"
+
 echo "Building in $tmpdir."
 cp -r . $tmpdir/
 
-pushd $tmpdir
```

Comment:
Is there any specific reason why we are removing `pushd/pop`? Is it because of sh instead bash?

Reply:
yes. it can be restored as we are not using the docker;git image anymore anyways.
what was the motivation/pro to use popd/pushd?